_content/booklets/satipatthanavibhango_anandajoti.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Satipaṭṭhānavibhaṅgo (Vibh. 7)
+Analysis of the Ways of Attending to Mindfulness"
+authors:
+  - anandajoti
+external_url: "https://www.ancient-buddhist-texts.net/Texts-and-Translations/Satipatthanavibhanga/Satipatthanavibhanga.pdf"
```

Comment:
Remove the `Sati...pdf` off the end to link to this work's main page rather than to the PDF.